<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseScreen</class>
 <widget class="QWidget" name="PauseScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="leftSpacing">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusBarrier" name="focusBarrierTop">
         <property name="text">
          <string notr="true">Barrier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="resumeButton">
         <property name="text">
          <string>Resume</string>
         </property>
         <property name="description">
          <string>Resume the game</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButton_2">
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="description">
          <string>Go to Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="mainMenuButton">
         <property name="text">
          <string>Main Menu</string>
         </property>
         <property name="description">
          <string>Abandon this game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="FocusBarrier" name="focusBarrierBottom">
         <property name="text">
          <string notr="true">Barrier</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="GamepadLabel" name="AHudButton">
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GamepadLabel" name="BHudButton">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamepadLabel</class>
   <extends>QLabel</extends>
   <header location="global">gamepadlabel.h</header>
  </customwidget>
  <customwidget>
   <class>FocusBarrier</class>
   <extends>QPushButton</extends>
   <header location="global">focusbarrier.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
